docs/admin/rescheduler.md

```diff
@@ -0,0 +1,51 @@
+---
+assignees:
+- davidopp
+- filipg
+- piosz
+
+---
+
+* TOC
+{:toc}
+
```

Comment:
This doc needs a title. How about

`# "Guaranteed" scheduling of critical addon pods"`
?